.github/workflows/connector_checklist.yml

```diff
@@ -0,0 +1,23 @@
+name: Add Connector Merge Checklist
+on:
+  pull_request:
+    types: [opened]
+    paths:
+      - "airbyte-integrations/connectors/source-**"
+      - "airbyte-integrations/connectors/destination-**"
+jobs:
+  checklist_job:
+    name: Add Connector Merge Checklist Job
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Checklist
+        uses: wyozi/contextual-qa-checklist-action@master
+        with:
+          comment-header: "### Before Merging a Connector Pull Request \n\n Wow! What a great pull request you have here! 🎉 \n\n To merge this Connector please ensure the following has been done/considered: \n\n"
+          comment-footer: "If the checklist is complete, but the CI check is failing, toggle the `checklist-action-run` on/off to re-run the checklist."
+          show-paths: false
+          input-file: airbyte-ci/connectors/CONNECTOR_CHECKLIST.yaml
+          gh-token: ${{ secrets.GITHUB_TOKEN }}
+          # See options documentation below
```

Comment:
```suggestion
```
Nit: either remove, or link to the documentation if you want to know there are available options